server/server/controllers/Restaurant.js

```diff
@@ -198,9 +198,11 @@ async function removeMenuItem(req, res) {
 }
 
 async function getRestaurantInfo(req, res) {
+  const token = req.headers.authorization;
+  const {restaurantId} = req.params;
   try {
-    const { restaurantId } = req.params;
-    const restaurantInfo = await restaurantSchema.findbyId(restaurantId);
+    jwt.verify(token, process.env.JWT_SECRET);
```

Comment:
Modify this to be an extraction of the token to be used to ensure that the token owner is accessing their own restuarant